File: src/BubbleSort.cpp
#include"BubbleSort.h"

namespace chain {
    void BubbleSort::Sort(int &dist1, int &dist2, int &dist3) {
        double temp;

        if (dist1 > dist2) {
            temp = dist1;
            dist1 = dist2;
            dist2 = temp;
        }
        if (dist2 > dist3) {
            temp = dist2;
            dist2 = dist3;
            dist3 = temp;
        }
        if (dist1 > dist2) {
            temp = dist1;
            dist1 = dist2;
            dist2 = temp;
        }
    }
    BubbleSort::~BubbleSort(){}
}

File: BubbleSort.py
#!/usr/bin/python3

class BubbleSort():
    def Sort(self, dist):
        if (dist[0] > dist[1]):
            temp = dist[0]
            dist[0] = dist[1]
            dist[1] = temp
        if (dist[1] > dist[2]):
            temp = dist[1]
            dist[1] = dist[2]
            dist[2] = temp
        if (dist[0] > dist[1]):
            temp = dist[0]
            dist[0] = dist[1]
            dist[1] = temp
